### ISSUE TITLE
Chinese encoding problem in v1.1.1

### ISSUE THREAD

**Issue by Vzz1c** [gerardog/gsudo#129]
Compared to 1.1.0 there is a serious error, the cmd window displayed after calling gsudo command is [Administrator], the font in the window is UTF-8, and my script files are written according to the local default encoding GBK, so the display will be garbled, in fact the global UTF-8 encoding there are many problems (windows There are special settings, and has been experimental), and I right-click the script to open the window with administrator privileges prompted [管理员] (I use Chinese language), so the window encoding is GBK, 1.1.0 version does not have this problem, so I temporarily down to this version to use, I hope to quickly fix, thank you!

Translated with www.DeepL.com/Translator (free version)

**Comment by gerardog** [comment]
Hi!
Writing international applications is hard. 

In v1.1.1, I introduced a fix for other problem. I forced UTF-8 console. I incorrectly assumed unicode would work on all international scenarios. That fix is the cause of your problem. 
Nonetheless, I don't understand completely all your symptoms. The more information I have, the better I could ensure I can fix this problem (and the original one).
Which windows do you use? Chinese edition?
Could you please upload a few screenshots comparing v1.1.0 with v1.1.1 ? 
Also, please open CMD, run `chcp`, then Run gsudo, and run `chcp` again.
That would be super helpful.

So far, I got:
* Window title changed: 管理员 vs Administrator
* Cmd or Powershell with different encoding: Scripts interpreted incorrectly 

Anyone else having this problem, please jump in.

Thanks

**Comment by Vzz1c** [comment]
![image](https://user-images.githubusercontent.com/49944812/159246546-50b6c346-b435-4119-9564-2d4a0c026555.png)

Global UTF-8 is currently not possible, because the default code of cmd and powershell is ANSI code, used to apply the language standards of different countries, such as Chinese ANSI corresponds to GBK, Japanese ANSI corresponds to JIS, the code of different countries are not compatible, the problem is not UTF-8 code itself will lead to garbled code, but I open the cmd or powershell window default encoding is GBK, if the code written in UTF-8 involves Chinese will lead to garbled code, (other countries should be the same problem), and to modify the default value needs to be manually changed through the registry, I set the global UTF-8 encoding a long time ago, the window of many text garbled, the software also has many problems I guess this problem is not necessarily a problem of encoding settings, it seems to be a problem caused by opening the cmd with the permission of the built-in account Administrator account? I do not understand these, this is my guess

Translated with www.DeepL.com/Translator (free version)

**Comment by gerardog** [comment]
I configured a virtual machine and installed GBK as default language and now I have seen what the problem is.

I learned that changing the "current process" encoding inside one app (For example `Console.InputEncoding = System.Text.Encoding.UTF8`), also changes the console-host code page, and that change stays even after the app the closed. 

Also the whole console window changes its font and language the moment the codepage is changed to 65001 (utf-8). CMD language changes, and what encoding is assumed for files without BOM... 

So, I decided that `gsudo` should not change the codepage. If an encoding problem occurs, that should be addressed by each user... English users are safe to change to utf-8 without consequences, but others don't.
